### PR TITLE
#11272 Throw an exception early when invalid task is loaded

### PR DESCRIPTION
**Pull request by burzum**
See #11272

### CODE REVIEW

**Review by dereuromark — APPROVED** [review]
This sure helps console development! thank you.

File: src/Console/Shell.php [hunk 1]
```diff
@@ -197,6 +198,8 @@ public function __construct(ConsoleIo $io = null)
         if (isset($this->modelClass)) {
             $this->loadModel();
         }
+
+        $this->loadTasks();
```
Comment (markstory) [inline src/Console/Shell.php]: Why does this need to be done earlier? Wouldn't doing this in `initialize()` still be early enough?

Comment (dereuromark) [reply src/Console/Shell.php]: I guess it would be.

Comment (burzum) [reply src/Console/Shell.php]: I haven't realized it's already been called there. Change the code. I've also refactored the added code a little and put it in it's own method to allow easier overriding and reduce method complexity.

**Comment by markstory** [comment]
Nice work @burzum

**Pull request merged**